dotcom-rendering/src/client/ophan/ophan.ts

```diff
@@ -143,3 +149,17 @@ export const recordPerformance = async (): Promise<void> => {
 		performance,
 	});
 };
+
+const experiencesSet = new Set(['dotcom-rendering']);
```

Comment:
Can we assume [`Set` is supported well enough](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#browser_compatibility)?